### PR TITLE
adding comparison table

### PR DESCRIPTION
**Pull request by danielcbright**
The idea here is to get a good vision of what the end-game looks like, and that's a full move to effortless for config + hab managed apps, this table is a start at figuring out what that looks like, my thoughts are a bit scattered so please correct and make changes :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,24 @@ nil: {"nil"=>41, "redhat_7.2"=>5}
...
+
+## Comparison - Chef Infra | Effortless
+
+|Feature|Chef Infra|Effortless|
+|-------|----------|----------|
+|**chef-client upgrade**|[Chef Client Updater Cookbook](https://github.com/chef-cookbooks/chef_client_updater)|[chef/chef-client Hab Package](https://bldr.habitat.sh/#/pkgs/chef/chef-client/latest)|
+|**Organizations**|[Chef Server Orgs](https://docs.chef.io/server_orgs.html)|[Habitat Origins](https://www.habitat.sh/docs/using-builder/#builder-origin)|
+|**Environments**|[Chef Environments](https://docs.chef.io/environments.html)|[Habitat Channels](https://www.habitat.sh/docs/using-habitat/#continuous-deployment-using-channels)|
+|**Roles**|stored in `*.json`, hosted in Chef Infra Server|*see "Role Cookbooks"|
+|**Role Cookbooks**|[Role Cookbook Model](https://github.com/chef-cft/chef-examples/blob/master/examples/RoleCookbookModel.md)|Roles and Role Cookbooks will migrate to a [Habitat Channel](https://www.habitat.sh/docs/using-habitat/#continuous-deployment-using-channels) for each unique endpoint|
```
Comment (ncerny) [inline README.md:47]: Roles won't necessarily translate to channels.  This can be solved multiple ways.

File: README.md [hunk 2]
```diff
@@ -34,3 +34,24 @@ nil: {"nil"=>41, "redhat_7.2"=>5}
...
+|**Data Bags**|stored in `*.json`, hosted in Chef Infra Server||
```
Comment (ncerny) [inline README.md:48]: Databags can also live in effortless, but is probably an antipattern

File: README.md [hunk 3]
```diff
@@ -34,3 +34,24 @@ nil: {"nil"=>41, "redhat_7.2"=>5}
...
+|**Environment-level Attributes**|stored in `*.json`, hosted in Chef Infra Server||
```
Comment (ncerny) [inline README.md:49]: Environment-level attributes translate to attributes.json which is generated from habitat config (user.toml or other config).

**Review by ncerny — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielcbright) [reply README.md:47]: I'm going to resolve all of these conversations, I think these are good points to start iterating on.

**Pull request merged**